crates/biome_js_analyze/src/assist/source/organize_imports.rs

```diff
@@ -1,143 +1,328 @@
-    /// import uncle from "../uncle";
-    /// import sibling from "./sibling";
-    /// import imageUrl from "url:./image.png";
+    /// Chunks also ends as soon as a statement or a side-effect import (also called bare import) is encountered.
+    /// The following example contains three chunks.
+    ///
+    /// ```js,ignore
+    /// import A from "a";
+    /// import "x";
+    /// import * as B from "b";
+    /// function f() {}
+    /// import * as C from "c";
     /// ```
     ///
-    /// You can apply the sorting in two ways: via [CLI](#import-sorting-via-cli) or [VSCode extension](#import-sorting-via-vscode-extension).
+    /// Finally, a last way of intentionally start a new chunk is to use a detached comment.
+    /// A **detached comment** is a comment separated of an import or export with a blank line.
+    /// Comments that are not separated of an import or export by a blank line are **attached comments**.
+    /// The following example contains a detached comment that split the imports into two chunks.
     ///
-    /// ## Grouped imports
+    /// ```js,ignore
+    /// // Attached comment 1
+    /// import A from "a";
+    /// // Attached comment 2
+    /// import * as B from "b";
     ///
-    /// It's widespread to have import statements in a certain order, primarily when you work on a frontend project, and you import CSS files:
+    /// // Detached comment
     ///
-    /// ```js
-    /// import "../styles/reset.css";
-    /// import "../styles/layout.css";
-    /// import { Grid } from "../components/Grid.jsx";
+    /// import * as C from "c";
     /// ```
     ///
-    /// Another common case is import polyfills or shim files, that needs to stay at the top file:
+    /// The import and export organizer ensures that chunks are separated from each other with a blank lines.
+    /// Only side-effect imports adjacent to a chunk are not separated by a blank line.
+    ///
+    /// Imports or exports of a chunk are sorted together.
+    /// Import and export sorting uses several layer of order.
+    ///
+    /// ## Import and export groups
+    ///
+    /// Imports or exports of a chunk are divided into groups.
+    /// By default, it exists a single group: the implicit group.
+    ///
+    /// You can define extra groups using the `groups` option.
+    /// The organizer provides predefined groups and allows you to create new groups based on a glob or a list of globs.
+    /// The predefined groups are:
     ///
-    /// ```js
-    /// import "../polyfills/array/flatMap";
-    /// import { functionThatUsesFlatMap } from "./utils.js";
+    /// - `:NODE:`: all imports and exports with a source starting by the protocol `node:` or that corresponds to a builtin Node,js module such as `path`
+    /// - `:BUN:`: all imports and exports with a source starting by the protocol `bun:`
+    ///
+    /// Given the following configuration and code, the import organizer propose to reorder the imports to place the Node.js imports first.
```

Comment:
I don't think we should show any option here, because we were talking about implicit/predefined groups. An option isn't needed. However, I suppose you want to show the options **if users want to change the order**. If so, it shouldn't be in this section, because we're talking about the default order.

Reply:
Groups affects how we sort imports. Imports are divided just after chunking. This seems a logical step to talk about them after chunks and before the order that is applied inside a group.

Reply:
I probably didn't explain myself. Talking about groups is totally. What I don't understand is the use of an example *with options* if we're talking about **defaults**.